### PR TITLE
Update hybrid-modern-auth-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Added signin options registry key to prerequisites

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6900

File: microsoft-365/enterprise/hybrid-modern-auth-overview.md [hunk 1]
```diff
@@ -165,6 +165,7 @@ Verify and check these items off your list before you continue:
     > Hybrid modern authentication is not supported with the [Hybrid Agent](/exchange/hybrid-deployment/hybrid-agent).
 
   - Make sure both an on-premises test user, as well as a hybrid test user homed in Office 365, can login to the Skype for Business desktop client (if you want to use modern authentication with Skype) and Microsoft Outlook (if you want to use modern authentication with Exchange).
+  - Make sure the SignInOptions setting in Microsoft Office is not configured to its most restrictive setting. For more information, see [ How to Allow office to connect to the internet](/office365/troubleshoot/access-management/office-feature-disabled)
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/hybrid-modern-auth-overview.md]:
```suggestion
  - Make sure the SignInOptions setting in Microsoft Office is not configured to its most restrictive setting. For more information, see [How to allow Office to connect to the internet](/office365/troubleshoot/access-management/office-feature-disabled).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kelleyvice-msft Gentle reminder. Thanks

**Pull request merged**